inline_svg.gemspec

```diff
@@ -18,13 +17,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec", "~> 3.2"
-  spec.add_development_dependency "rspec_junit_formatter", "0.2.2"
-  spec.add_development_dependency "pry"
-  spec.add_development_dependency "rubocop"
+  spec.metadata['rubygems_mfa_required'] = 'true'
+  spec.required_ruby_version = '>= 3.0'
```

Comment:
3.0 is also in EOL, this can start from 3.1

Reply:
@jamesmartin do you want to drop 3.0 and start from 3.1?

Reply:
@tagliala yes, thanks for the suggestion, let's start with 3.1. 👍